framework/SwiftPlot/HelperFunction.swift

```diff
@@ -67,25 +67,33 @@ public func lerp(startColor: Color, endColor: Color, _ t: Float) -> Color {
     return Color(r, g, b, a)
 }
 
-public func rotatePoint(point: Point, center: Point, angleDegrees: Float) -> Point{
+public func rotatePoint(point: Pair<FloatConvertible,FloatConvertible>,
+                        center: Pair<FloatConvertible,FloatConvertible>,
+                        angleDegrees: Float) -> Pair<FloatConvertible,FloatConvertible>{
```

Comment:
When you're defining Pair in this manner, I believe you're erasing the type. I think instead you'll want to use Pair<T, U> where T: FloatConvertible, U: FloatConvertible. That preserves the type, but makes sure they conform to the protocol, where right now you're using the protocol as the type.